roles/satellite-clone/tasks/main.yml

```
@@ -319,10 +319,18 @@
 
   - include_tasks: reset_pulp_data.yml
     when:
+      - reset_pulp_data
       - not clone_pulp_data_exists
       - not online_backup
       - satellite_version in ["6.6", "6.7", "6.8", "6.9"]
 
+  - include_tasks: reset_pulpcore_data.yml
+    when:
+      - reset_pulp_data
+      - not clone_pulp_data_exists
+      - not online_backup
+      - satellite_version in ["6.10"]
```

Elsewhere you mention 7.0 specific actions, but this would prevent that. Should this be ready for the future and say
```suggestion
      - satellite_version not in ["6.6", "6.7", "6.8", "6.9"]
```